Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    /// <summary>
+    /// API for working with MixedRealityToolkit folders contained in the project.
+    /// </summary>
+    [InitializeOnLoad]
+    public static class MixedRealityToolkitFiles
+    {
+        private class AssetPostprocessor : UnityEditor.AssetPostprocessor
```

Comment:
>private class AssetPostprocessor : UnityEditor.AssetPostprocessor [](start = 8, length = 65)

Nit: It might be helpful to have a doccomment like:

/// <summary>
/// A helper class that ensures that the list of mrtkFolders is updated as assets are imported,
/// deleted, or moved.
/// </summary>

Reply:
I was debating, it's private cause it only serves the purpose of being an AssetImporter; will add this comment.

Reply:
Yep, could also not be a doccomment (i.e. just a regular comment). Whether or not it shows up in docfx generated stuff is of questionable value (because it's private as you say)